src/elements/hero/src/index.tsx

```diff
@@ -55,92 +56,97 @@ const Hero: React.FC<Props> = ({
 
   return (
     <div>
-      <div
-        sx={{
-          position: 'relative',
-          overflow: 'hidden',
-          variant: 'elements.hero.wrapper',
-          backgroundColor: customBgColor || undefined
-        }}
-        className={className}
+      <Palette
+        as="div"
+        sx={{ variant: 'elements.hero.wrapper', overflow: 'hidden' }}
+        px={{ backgroundColor: 'backgroundColor' }}
```

Comment:
your setting backgroundColor to a string called backgroundColor is that intentional?

Reply:
Yep, it's tied to the Palette 'backgroundColor' field in Contentful